components/home/homepage-menu.tsx

```diff
@@ -19,11 +19,22 @@ export function HomePageMenu() {
               className: "h-6 w-6 text-brand-500",
             })}
           </div>
-          <Link href={item.href}>
-            <a className="text-gray-900 font-semibold text-xs mt-3 helper-link-cover">
+          {item.external ? (
+            <a
+              className="text-gray-900 font-semibold text-xs mt-3 helper-link-cover cursor-pointer"
```

Comment:
I suggest to just remove the `cursor-pointer`

Reply:
ah I see, 
as long as tag `<a>` have attribute `href` it will already has cursor CSS

![image](https://user-images.githubusercontent.com/1397612/129211024-5af4232f-b764-4c44-b558-60b30f4a248e.png)